nssp_demo/forecast_state.py

```diff
@@ -87,12 +102,34 @@ def main(
 
     logger.info(f"last training date: {last_training_date}")
 
+    # +1 because max date in dataset is report_date - 1
     first_training_date = last_training_date - timedelta(
         days=n_training_days - 1
     )
 
-    datafile = f"{report_date}.parquet"
-    nssp_data = pl.scan_parquet(Path(nssp_data_dir, datafile))
+    logger.info(f"First training date {first_training_date}")
+
+    facility_level_nssp_data, state_level_nssp_data = None, None
+
+    if report_date in available_facility_level_reports:
+        logger.info(
+            "Facility level data available for " "the given report date"
+        )
+        facility_datafile = f"{report_date}.parquet"
+        facility_level_nssp_data = pl.scan_parquet(
+            Path(facility_level_nssp_data_dir, facility_datafile)
+        )
+    if state_report_date in available_state_level_reports:
```

Comment:
This is always true because of https://github.com/CDCgov/pyrenew-hew/pull/87/files#r1823063547. Is that intended?

Reply:
Improved the logic so that it's no longer true (in particular if you try to grab data from _before_ the state level report dates start.

Reply:
Longer run we should perhaps refactor and generalize this data hybridization and date picking structure, but I'm reluctant to spend that time now given that other things may change.